.github/workflows/upload-cloud-images.yaml

```diff
@@ -240,5 +240,8 @@ jobs:
         with:
           azcliversion: latest
           inlineScript: |
+            # Install openssh-clients to be able to use ssh-keygen
+            tdnf install -y openssh-clients
```

Comment:
tdnf is correct???

Reply:
I didn't know that either, had to google it up